src/decorator-options/ControllerOptions.ts

```diff
@@ -0,0 +1,8 @@
+import {TransformationOptions} from "../TransformationOptions";
+
+/**
+ * Extra options that apply to each controller action.
+ */
+export interface ControllerOptions extends TransformationOptions {
```

Comment:
I think that you should implement `TransformationOptions`, not extending it - it would be ready for future extension/changes because you can't extend multiple interfaces.